### ISSUE TITLE
With Pretrained Model got nonsense answers.

### ISSUE THREAD

**Issue by Edward-liang** [Conchylicultor/DeepQA#85]
I followed the instructions in **Readme** file to deploy the pretrained model and nltk corpus under certain folders. 
Env: Python3.6.0 Tensorflow1.0 
The program goes smoothly with "python main.py --modelTag pretrainedv2 --test interactive", However I got nonsense talks as followed.
Has any one met similar problems before, please?

> 
> Q: hello
> A: Hepburn reiss reiss reiss reiss qualen qualen qualen qualen shrouds shrouds shroud
> 
> Q: Hi.
> A: Hepburn hepburn daylight foresee dedicated dedicated dedicated dedicated mugshot m
> 
> Q: What is your name?
> A: Megan gracious-living-wise gracious-living-wise gracious-living-wise rotting rotti ng rotting rotting rotting rotting connelly connelly
> 
> Q: What does that mean ?
> A: Shitty shitty shitty shitty rotting rotting rotting aspiration aspiration aspirati on aspiration aspiration
> 
> Q: How old are you?
> A: Aurelius aurelius lindenmeyer chargin chargin chargin conscientious conscientious conscientious ripper ripper ripper
> 
> Q: Will Google hire me?
> A: Starbuck gracious-living-wise gracious-living-wise berlin crumble-crumble crumble-crumble crumble-crumble crumble-crumble crumble-crumble crumble-crumble crumble-crumble crumble-crumble
> 
> Q: Tell me the alphabet.
> A: Guiding cuban-americans decompose decompose redistribution redistribution erin erin erin erin erin erin
> 
> Q: That's not nice.
> A: Slashed slashed slashed slashed slashed traidor traidor blue-winged blue-winged blue-winged blue-winged blue-winged
>

**Comment by Conchylicultor** [comment]
Could you copy the full terminal output please ?

**Comment by Edward-liang** [comment]
@Conchylicultor Thank you very much for your reply. I find a hint says:
No previous model found, starting from clean directory: /home/zl/DeepQA-master/save/model-pretrainedv2
Testing: Launch interactive mode:

which was caused by my wrong handling of the model: I just extracted the contents in the zip "model-pretrainedv2" which their parent folder. Now I move them under model-pretrainedv2 folder, and everything now goes fine. 

Again, thank you for your timely reply.
